cinema_service/settings.py

```diff
@@ -20,12 +22,10 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.getenv("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.getenv("DEBUG") != "False"
 
 ALLOWED_HOSTS = []
```

Comment:
In the production environment, it's important to define the ALLOWED_HOSTS setting. This is a security measure to prevent HTTP Host header attacks. You could load it from an environment variable, just like you did with SECRET_KEY and DEBUG.